app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -48,8 +48,10 @@ export const ContentListView: React.FC = () => {
 
   React.useEffect(() => {
     // Extract query string values and place them into redux store.
-    storeFilter(queryToFilter(filter, window.location.search));
-    storeFilterAdvanced(queryToFilterAdvanced(filterAdvanced, window.location.search));
+    if (!!window.location.search) {
```

Comment:
Only do this if there are query parameters.  This however may result in future issues if a query parameter is specified for a different reason.  There appears to be a fundamental flaw in the implementation to extract query parameters to influence state.